<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Excel读写</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Word读写</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>291</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="InExcelLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>我爱最新的Qt 5.12</string>
     </property>
    </widget>
    <widget class="QPushButton" name="writeExcelPushButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>写入</string>
     </property>
    </widget>
    <widget class="QPushButton" name="readExcelPushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>读出</string>
     </property>
    </widget>
    <widget class="QLabel" name="OutExcelLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>291</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="InWordLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>我爱最新的Qt 5.12</string>
     </property>
    </widget>
    <widget class="QPushButton" name="writeWordPushButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>写入</string>
     </property>
    </widget>
    <widget class="QPushButton" name="readWordPushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>读出</string>
     </property>
    </widget>
    <widget class="QLabel" name="OutWordLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>329</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
